projects/RabbitMQ.Client/client/impl/ModelBase.cs

```diff
@@ -625,24 +572,7 @@ public void HandleBasicConsumeOk(string consumerTag)
             ReadOnlyMemory<byte> body,
             byte[] rentedArray)
         {
-            IBasicConsumer consumer;
-            lock (_consumers)
-            {
-                consumer = _consumers[consumerTag];
-            }
-            if (consumer is null)
-            {
-                if (DefaultConsumer is null)
```

Comment:
@bollhals Remember to forward DefaultConsumer to Dispatcher

Reply:
Does this mean that there's more work left to be done as part of this PR?

Reply:
Yes, with the move of the default consumer I forgot to forward the default consumer to the dispatcher. This also means that the test coverage is lacking as well in this area. (I either take a look ig its easy or create a new issue for tracking

Reply:
fixed but there are no uses of DefaultConsumer in the whole code... Is it even still needed?

Reply:
It is meant to be subclassed by applications.

Reply:
I don't unsterstand what you've meant by subclass in this context.